test.sh

```diff
@@ -113,3 +113,7 @@ do
   echo "Pass."
 done
 echo "::endgroup::"
+
+echo "::group::Ensure cleanup crons are working ..."
+docker-compose ps | grep -q -- "-cleanup_.\+[[:space:]]\+Up[[:space:]]\+"
+echo "::endgroup::"
```

Comment:
Yay for test case! 😍

Reply:
Appears to pass, even. :)

https://github.com/getsentry/onpremise/runs/1904387743?check_suite_focus=true#step:4:1946